src/mobu/business/jupyterloginloop.py

```diff
@@ -31,10 +30,13 @@ class JupyterLoginLoop(Business):
     """
 
     def __init__(
-        self, logger: BoundLogger, options: Dict[str, Any], user: User
+        self,
+        logger: BoundLogger,
+        business_config: BusinessConfig,
+        user: AuthenticatedUser,
     ) -> None:
-        super().__init__(logger, options, user)
-        self._client = JupyterClient(user, logger, options)
+        super().__init__(logger, business_config, user)
+        self._client = JupyterClient(user, logger, business_config)
```

Comment:
Something I've been curious about reading your code; usually when doing parameters, even if they can be positional, I name them.  Am I doing it wrong, or is this simply a matter or personal preference?

Reply:
It's a matter of personal taste. I tend to use the rule of three: if there are more than three parameters, they should probably be named.